homeassistant/components/lifx/coordinator.py

```diff
@@ -199,3 +210,12 @@ async def async_set_hev_cycle_state(self, enable: bool, duration: int = 0) -> No
             await async_execute_lifx(
                 partial(self.device.set_hev_cycle, enable=enable, duration=duration)
             )
+
+    def async_current_infrared_brightness(self) -> str | None:
+        """Return the current infrared brightness as a string."""
+        return infrared_brightness_value_to_option(self.device.infrared_brightness)
```

Comment:
```suggestion
    @property
    def current_infrared_brightness(self) -> str | None:
        """Return the current infrared brightness as a string."""
        return infrared_brightness_value_to_option(self.device.infrared_brightness)
```

Let's make this a property so its clear its not expected to do I/O